test/integration/eu-west-1/k8s-qa18/services/terragrunt.hcl

```diff
@@ -74,4 +73,21 @@ inputs = {
 
   kiam_chart_version = "5.9.0" # 5.10.0 includes https://github.com/uswitch/kiam/pull/427, but image does not - breaks WhitelistRouteRegexp/AllowRouteRegexp
 
+  # --------------------------------------------------
+  # Monitoring namespace
+  # --------------------------------------------------
+
+  monitoring_namespace_deploy = true
```

Comment:
monitoring_namespace_iam_roles not set as per production

Reply:
Because it is not needed here for now. It defaults to empty string.

Reply:
This makes it so retarded that we set default chart versions to null then.